base_time_window/models/time_window_mixin.py

```diff
@@ -31,18 +31,16 @@ def check_window_no_overlaps(self):
         for record in self:
             if record.time_window_start > record.time_window_end:
                 raise ValidationError(
-                    _("%(end_time)s must be > %(start_time)s")
-                    % (
-                        {
-                            "end_time": self.float_to_time_repr(record.time_window_end),
-                            "start_time": self.float_to_time_repr(
-                                record.time_window_start
-                            ),
-                        }
+                    self.env._(
+                        "%s must be > %s",
+                        self.float_to_time_repr(record.time_window_end),
+                        self.float_to_time_repr(record.time_window_start),
```

Comment:
We should keep placeholders as their places could be different depending on the language/translation.